src/parser/parse_tree_cl.c

```diff
@@ -14080,10 +14086,16 @@ pt_print_select (PARSER_CONTEXT * parser, PT_NODE * p)
 	  if (p->info.query.q.select.hint & PT_HINT_ORDERED)
 	    {
 	      /* force join left-to-right */
-	      q = pt_append_nulstring (parser, q, "ORDERED");
-	      if (p->info.query.q.select.ordered)
+	      q = pt_append_nulstring (parser, q, "ORDERED ");
+	    }
+
+	  if (p->info.query.q.select.hint & PT_HINT_LEADING)
```

Comment:
If PT_HINT_ORDERED and PT_HINT_LEADING are printed together, it is difficult to know which hint was used.
```suggestion
	  else if (p->info.query.q.select.hint & PT_HINT_LEADING)
```
Please see the example.
```
drop table if exists t1, t2, t3, t4;

create table t1 (c1 int);
create table t2 (c1 int);
create table t3 (c1 int, c2 int);
create table t4 (c1 int, c2 int);

insert into t1 values (1);
insert into t2 values (1);
insert into t3 values (1, 2);
insert into t4 values (1, 2);

select /*+ recompile ordered leading(t4,t1) */ count (*) from t1, t2, t3, t4 where t1.c1 = t2.c1 and t2.c1 = t3.c1 and t3.c2 = t4.c2;

/*
...
Query stmt:

select /*+ ORDERED LEADING(t4, t1) */ count(*) from t1 t1, t2 t2, t3 t3, t4 t4 where t1.c1=t2.c1 and t2.c1=t3.c1 and t3.c2=t4.c2
*/
```

Reply:
```
If you specify the ORDERED hint, all LEADING hint is ignored. 
```
I think it would be okay to add the above information to the manual.